apps/prairielearn/elements/pl-dropdown/pl-dropdown.mustache

```diff
@@ -42,7 +42,7 @@
             <span class="badge text-danger badge-invalid"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid</span>
             <button id="pl-dropdown-popover-button-{{uuid}}" class="btn btn-sm btn-secondary small border ml-1"
                     data-placement="auto" data-trigger="focus" data-toggle="popover"
-                    data-html="true" title="Format Error" data-container="body"
+                    data-html="true" title="Format Error" tabindex="0" data-container="body"
```

Comment:
Why this change?

Reply:
This fixes a display bug for dropdowns where the format error popup wouldn't appear if you hover over the button. It's technically independent of the other changes, so I can factor it into a separate pull request if you prefer.

Reply:
For context: this popup wouldn't appear if you hover over the "Why?" button. It works for other elements because those already have the tabindex="0" attribute.

<img width="417" alt="Screenshot 2024-07-26 at 4 42 56 PM" src="https://github.com/user-attachments/assets/caf1c550-4c18-442d-a81a-c2217896d0c5">

Reply:
Given that we've specified `data-trigger="focus"`, I wouldn't expect it to show on hover. Can you say more about why you think this is a bug, e.g. do other elements display on hover?

Reply:
The format error is not intended to be viewable using hover, you need to focus (e.g., click or tab into) to trigger it to open. This is consistent with other elements.

Reply:
Sorry, I actually meant clicking/focusing the element. For me, the previous version does not display the pop-up at all (neither when clicking nor when tabbing). I just tried different browsers and this seems to be a Safari-specific bug. 

Similar elements, like pl-matching, already have the tabindex="0" attribute and work correctly in Safari, as does pl-dropdown if I add tabindex="0" for it as well.

Reply:
Fair enough, Bootstrap also documents that `tabindex="0"` is required for the `focus` trigger to work correctly in all browsers:

https://getbootstrap.com/docs/5.3/components/popovers/#dismiss-on-next-click

#6272 is the right long-term move, but I won't ask for you to make that change here.